device/core/src/device_client.ts

```diff
@@ -21,9 +25,14 @@ import { Twin  } from './twin';
  * or {@link azure-iot-device.Client.fromSharedAccessSignature|fromSharedAccessSignature}
  * to create an IoT Hub device client.
  */
-export class Client extends EventEmitter {
-  private _internalClient: InternalClient;
-
+export class Client extends InternalClient {
+  private _c2dEnabled: boolean;
+  private _deviceDisconnectHandler: (err?: Error, result?: any) => void;
+  /**
+   * @private
```

Comment:
>* @private [](start = 3, length = 10)

nit: I think the docs.microsoft.com jsdoc parsers are pretty good about honoring the private scoping on the variable.  From what I've seen, they're pretty good about pulling everything (like types and parameters) out of .ts function prototypes, so most of the duplication that you have to put in jsdoc for vanilla javascript isn't necessary in typescript.